.github/ISSUE_TEMPLATE/installation_problem.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Installation problem
+about: Get help installing or starting TensorBoard
```

Comment:
Do we want to phrase this as "get help"?  I think it might be better to stick to the clear messaging that our issue tracker should be for real bugs and feature requests, not for general assistance.  Installation issues in particular are often specific to the user, so it's hard to scale supporting those types of concerns.